roles/openshift_sanitize_inventory/tasks/deprecations.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Check for usage of deprecated variables
+  debug: msg="{{ __deprecation_header }} {{ item }} is a deprecated variable and will be no longer be used in the next minor release. Please update your inventory accordingly."
+  when:
+  - lookup('env', item) | bool
```

Comment:
I checked the negative case but not the positive case... I'll verify whether or not it can check an inventory var (hopefully it can, otherwise this approach wont work)